platform/default/default_file_source.cpp

```diff
@@ -189,8 +189,8 @@ DefaultFileSource::DefaultFileSource(const std::string& cachePath,
                                      std::unique_ptr<FileSource>&& assetFileSource_,
                                      uint64_t maximumCacheSize)
         : assetFileSource(std::move(assetFileSource_))
-        , thread(std::make_unique<util::Thread<Impl>>(util::ThreadContext{"DefaultFileSource", util::ThreadPriority::Low},
```

Comment:
All the `util::Thread` objects were setting priority to low, so I moved it to set to low on the `util::ThreadedObject` by default.